testing/rundeck/APKBUILD

```diff
@@ -0,0 +1,76 @@
+# Contributor: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+# Maintainer: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+pkgname=rundeck
+pkgver=2.7.2
+pkgrel=0
+pkgdesc="Job scheduler and runbook automation. Enable self-service access to existing scripts and tools."
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+url="http://rundeck.org"
+arch="all"
+license="ASL 2.0"
```

Comment:
but alpine suggests to use in this case `ASL 2.0` https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license